packages/app-utils/src/components/ConnectionBootstrap.tsx

```diff
@@ -32,7 +31,7 @@ export function ConnectionBootstrap({
   const client = useClient();
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const [error, setError] = useState<unknown>();
-  const [connection, setConnection] = useState<DhType.IdeConnection>();
+  const [connection, setConnection] = useState<dh.IdeConnection>();
   const [authFailedState, setIsAuthFailedState] = useState<boolean>(false);
```

Comment:
The state and setter should have consistent names. Also you don't need the type annotation as TS can infer it's a boolean.

```suggestion
  const [isAuthFailed, setIsAuthFailed] = useState(false);
```